ci/dockerfile/ubuntu22.04.dockerfile

```diff
@@ -97,14 +98,17 @@ RUN git clone https://github.com/giltene/wrk2.git \
 # NOTE about meson version: we support "0.56 or newer", so in CI we pin to latest patch version of
 # the earliest supported minor version (pip implicitly installs latest version satisfying the
 # specification)
+RUN python3 -m pip install --upgrade pip --user
```

Comment:
Same as above.

Reply:
We do need it for installing scikit packages. Without pip upgrade, it does not found the package